app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,3 @@
+class PostsController < ApplicationController
+  def index; end
+end
```

Comment:
- [ ] You still need to define the Post `#show` action as defined in the route above

  ```
  def show; end
  ```